src/coreclr/scripts/superpmi_benchmarks.py

```diff
@@ -0,0 +1,271 @@
+#!/usr/bin/env python3
+#
+# Licensed to the .NET Foundation under one or more agreements.
+# The .NET Foundation licenses this file to you under the MIT license.
+#
+#
+# Title: superpmi_benchmarks.py
+#
+# Notes:
+#
+# Script to perform the superpmi collection while executing the Microbenchmarks present
+# in https://github.com/dotnet/performance/tree/master/src/benchmarks/micro.
+
+import argparse
+import re
+import sys
+
+import stat
+from os import path
+from os.path import isfile
+from shutil import copyfile
+from coreclr_arguments import *
+from superpmi import ChangeDir, TempDir
```

Comment:
I'm not fond of anyone depending on superpmi.py; it makes it more fragile to change without potentially breaking (unknown) dependents.

Reply:
I agree. There are few methods in `super_setup.py` as well that I will extract in a `superpmi_utils.py` file in a future PR.